_includes/rest/users.md

```diff
@@ -613,3 +611,53 @@ For example, if you want the user with id `"3KmCvT7Zsb"` to have read and write
 ```
 
 If you want to access your data ignoring all ACLs, you can use the master key provided on the Dashboard. Instead of the `X-Parse-REST-API-Key` header, set the `X-Parse-Master-Key` header. For backward compatibility, you can also do master-level authentication using HTTP Basic Auth, passing the application id as the username and the master key as the password. For security, the master key should not be distributed to end users, but if you are running code in a trusted environment, feel free to use the master key for authentication.
+
+## Impersonating a user
+
+An application may have use cases where an operator or automated system needs to
+take action on a user's behalf, under the user's authority, without access to
+the user's credentials. The Parse REST API supports these use cases with the
+`/loginAs` endpoint. This endpoint takes a `userId` parameter, and must be
+called using the master key. It will create a session for the given user ID, and
+will return the same response format as the `/login` endpoint. The new session
+will have a `createdWith` value of `create`, and an `authProvider` value of
+`masterkey`.
+
+**The `/loginAs` endpoint performs no identity verification** - any client with
+the master key can log in as any user. You are strongly encouraged to apply
+careful security around any systems that call the `/loginAs` endpoint.
+
+<div class="language-toggle">
+<pre><code class="bash">
+curl -X GET \
+  -H "X-Parse-Application-Id: <span class="custom-parse-server-appid">${APPLICATION_ID}</span>" \
+  -H "X-Parse-REST-API-Key: <span class="custom-parse-server-restapikey">${REST_API_KEY}</span>" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "X-Parse-Revocable-Session: 1" \
+  -G \
+  --data-urlencode 'userId=abc123' \
+  <span class="custom-parse-server-protocol">https</span>://<span class="custom-parse-server-url">YOUR.PARSE-SERVER.HERE</span><span class="custom-parse-server-mount">/parse/</span>loginAs
+</code></pre>
+<pre><code class="python">
+import json,httplib,urllib
+connection = httplib.HTTPSConnection('<span class="custom-parse-server-url">YOUR.PARSE-SERVER.HERE</span>', 443)
+params = urllib.urlencode({"userId":"abc123"})
+connection.connect()
+connection.request('GET', '<span class="custom-parse-server-mount">/parse/</span>loginAs?%s' % params, '', {
+       "X-Parse-Application-Id": "<span class="custom-parse-server-appid">${APPLICATION_ID}</span>",
+       "X-Parse-REST-API-Key": "<span class="custom-parse-server-restapikey">${REST_API_KEY}</span>",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "X-Parse-Revocable-Session": "1"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+</code></pre>
+</div>
+
+At this time, the `/loginAs` endpoint does not run the `beforeLogin` or
+`afterLogin` hooks that would be invoked when calling the `/login` endpoint.
```

Comment:
Let's remove "at time time", because it isn't clear what this means. If _never_ invokes the triggers. Once this behavior changes, we'll update the docs.